### ISSUE TITLE
GUI

### ISSUE THREAD

**Issue by qsniyg** [qsniyg/ksp_stuff#22]
Although documentation on the configuration file format is definitely important, I think a GUI would be the easiest to use for most people.

Of course, this would be a relatively large project.

**Comment by Nanodragon999** [comment]
Just fyi, but making a gui for a project such as this could be easier depending on what you use:

- bash + gtk-dialog (so no need for C libs to make gtk guis)
- pythonqt (as it work better than the one for gtk, from my own experience)
- zenity (can be made using bash or python, anything else that could support it too)

**Comment by Nanodragon999** [comment]
You could also make a decent tui with just bash/python builtin too, although would be a bit offtopic in this case.

**Comment by qsniyg** [comment]
That's true, it would be possible... however, with the USVFS patch (that might actually be at least partially merged into wine soon), I don't think there's much worth to invest too much in this, as the proper solution is already (at least partially) done :)
